get_anndata.py

```diff
@@ -126,8 +126,14 @@ def make_pheno_cov(
     if fill_in_age:
         mean_age = sample_covs_df['age'].mean()
     cell_ind_df = expression_adata.obs.loc[
-        :, ['cell', 'individual', 'total_counts', 'sequencing_library']
+        :, ['cell', 'individual', 'total_counts', 'cohort', 'sequencing_library']
     ]
+    # make sequencing_library from categorical to dummy numerical covs
+    seq_lib_df = pd.get_dummies(cell_ind_df['sequencing_library'])
+    # do the same for cohort
+    cohort_df = pd.get_dummies(cell_ind_df['cohort'])
+    cell_ind_df = pd.concat([cell_ind_df, cohort_df, seq_lib_df], axis=1)
```

Comment:
Do you still want to leave the original  `sequencing_library` and `cohort` columns in the df, even if you won't use them?

Reply:
Yes I thought about it but I think they're clearer in their original form, and i always need to specify which covariates I want to test within the saige command so there's no risk they get included accidentally